lib/array.js

```diff
@@ -416,25 +443,17 @@ const find = (items, fn, done) => {
 //     err - <Error> | <null>
 //     result - <boolean>
 const every = (items, fn, done) => {
-  if (items.length === 0) {
-    done(null, true);
-    return;
-  }
-  let proceedItemsCount = 0;
-  const len = items.length;
-
-  const finish = (err, accepted) => {
-    if (!done) return;
-    if (err || !accepted) {
-      done(err, false);
-      done = null;
-      return;
+  findIndex(
+    items,
+    (item, callback) => fn(item, (err, accepted) => callback(err, !accepted)),
+    (err, index) => {
+      if (err || index !== -1) {
+        done(err, err ? undefined : false);
+        return;
+      }
+      done(null, true);
```

Comment:
```suggestion
      done(null, index !== -1);
```
And then it's possible to simplify the check upper to 
```
if (err) {
  done(err);
  return;
}
```